recovery/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,7 @@
     <string name="kinrecovery_restore">Restore</string>
     <string name="kinrecovery_backup">Backup</string>
 
-    <string name="kinrecovery_steps_format">%d/%d</string>
+    <string name="kinrecovery_steps_format" translatable="false">%d/%d</string>
```

Comment:
why the `translatable=false` is necessary.

Reply:
To remove the warning message from the IDE, they ask to translate this line to Portuguese.